### PR TITLE
UI Finish Homepage

### PR DESCRIPTION
**Pull request by datmaiz**

### CODE REVIEW

File: src/resources/stylesheets/pages/_home.scss [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+
+.container {
+  max-width: 1440px;
+  margin-inline: auto;
+}
+
+.banner-furniture {
+  @include center-vertical;
+  gap: 66px;
+  background-color: #e2d9d6;
```
Comment (xuanquocc) [inline src/resources/stylesheets/pages/_home.scss:11]: pls use golbal variable

File: src/resources/stylesheets/pages/_home.scss [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+    @include clamp(20px, 28px);
+    margin-bottom: 40px;
+  }
+
+  .view-details {
+    padding: 16px;
+  }
+
+  .introduce-item:first-child {
+    background-color: #fafafa;
```
Comment (xuanquocc) [inline src/resources/stylesheets/pages/_home.scss:63]: same thing above

**Review by xuanquocc — COMMENTED** [review]
color, font-weight, font-size let use global variable

File: src/resources/stylesheets/pages/_home.scss [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+  }
+
+  .desc {
+    @include font(18px, 400);
+    margin-top: 8px;
+  }
+
+  input[type="email"] {
+    width: 515px;
+    width: 100%;
```
Comment (xuanquocc) [inline src/resources/stylesheets/pages/_home.scss:340]: why do use width: 515px;
    width: 100%;

File: src/resources/stylesheets/pages/_home.scss (hunk 1 above)
Comment (datmaiz) [reply src/resources/stylesheets/pages/_home.scss:11]: I'll fix later, thanks

**Pull request merged**